From: Michael Webster <miketwebster@gmail.com>
Date: Mon, 7 Nov 2022 16:49:35 -0500
Subject: tiling: Remove unnecessary update_edge_constraints().

Both meta_window_maximize and meta_window_tile use
meta_window_maximize_internal(), and edge constraints are already
recalculated and updated there.

(cherry picked from commit ec17d19c76fdb53e5b0a96d3783403c7517abc26)

Forwarded: https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/2579
Forwarded: https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/2751
Origin: upstream, 43.3, commit:d93f5cf293863b11b326291cab2c79eb6871d0b6
---
 src/core/window.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/core/window.c b/src/core/window.c
index 7e1c35c..c5ee5af 100644
--- a/src/core/window.c
+++ b/src/core/window.c
@@ -2935,9 +2935,6 @@ meta_window_tile (MetaWindow   *window,
   meta_window_maximize_internal (window, directions, NULL);
   meta_display_update_tile_preview (window->display, FALSE);
 
-  /* Setup the edge constraints */
-  update_edge_constraints (window);
-
   if (!window->tile_match || window->tile_match != window->display->grab_window)
     {
       MetaRectangle old_frame_rect, old_buffer_rect;
